backend/api/Areas/Contacts/Models/Contact/OrganizationModel.cs

```diff
@@ -0,0 +1,45 @@
+namespace Pims.Api.Areas.Contact.Models.Contact
+{
+    /// <summary>
+    /// Provides a contact-oriented organization model.
+    /// </summary>
+    public class OrganizationModel
+    {
+        #region Properties
+        /// <summary>
+        /// get/set - The primary key to identify the organization.
+        /// </summary>
+        public long Id { get; set; }
```

Comment:
I don't see the corresponding **isDisabled** flag to organizations (they are in the DB). Shouldn't it be added to the model as well?

Reply:
They should, however this pr is only for individual

Reply:
I know but you added the Org model so assumed the change would be included. Not a bug deal - we just need to add it back when orgs are implemented. Hopefully we don't forget